PubnativeLite/PubnativeLite/AppMonet/AMInterstitialAdController.m

```diff
@@ -24,6 +24,7 @@
 
 @interface AMInterstitialAdController () <HyBidInterstitialAdDelegate>
 
+@property (nonatomic, strong) HyBidInterstitialAd *hybidInterstitialAd;
```

Comment:
`hyBidInterstitialAd` looks better. what do you think @faresbh?

Reply:
it's all about consistency 👍